apparmor.d/groups/freedesktop/polkit-kde-authentication-agent

```diff
@@ -34,8 +34,17 @@ profile polkit-kde-authentication-agent @{exec_path} flags=(attach_disconnected)
   /var/lib/dbus/machine-id r,
 
   owner @{user_config_dirs}/breezerc r,
+  owner @{user_config_dirs}/kdedefaults/plasmarc r,
 
   owner @{user_cache_dirs}/icon-cache.kcache rw,
+  owner @{user_cache_dirs}/polkit-kde-authentication-agent-@{int}/qmlcache/#@{int} rwl,
+  owner @{user_cache_dirs}/polkit-kde-authentication-agent-@{int}/qmlcache/*.qmlc{,.@{rand6}} rw,
+  owner @{user_cache_dirs}/polkit-kde-authentication-agent-@{int}/qmlcache/*.qmlc{,.@{rand6}} l -> @{user_cache_dirs}/polkit-kde-authentication-agent-@{int}/qmlcache/#@{int},
+  owner @{user_cache_dirs}/polkit-kde-authentication-agent-@{int}/qtpipelinecache-*/qqpc_opengl.lck rwk,
+  owner @{user_cache_dirs}/polkit-kde-authentication-agent-@{int}/qtpipelinecache-*/#@{int} rwl,
+  owner @{user_cache_dirs}/polkit-kde-authentication-agent-@{int}/qtpipelinecache-*/qqpc_opengl{,.@{rand6}} rw,
+  owner @{user_cache_dirs}/polkit-kde-authentication-agent-@{int}/qtpipelinecache-*/qqpc_opengl{,.@{rand6}} l -> @{user_cache_dirs}/polkit-kde-authentication-agent-@{int}/qtpipelinecache-*/#@{int},
```

Comment:
That should be simplified as:
```
 owner @{user_cache_dirs}/polkit-kde-authentication-agent-@{int}/ rw,
 owner @{user_cache_dirs}/polkit-kde-authentication-agent-@{int}/** rwlk ->  @{user_cache_dirs}/polkit-kde-authentication-agent-@{int}/**,
```

Reply:
Ah, so this is a general "trick" to combine an `rw` rule and a link rule into one?

Reply:
Yes, however, it may have some bug in some context. The following would be more bullet proof:
```
 owner @{user_cache_dirs}/polkit-kde-authentication-agent-@{int}/ rw,
 owner @{user_cache_dirs}/polkit-kde-authentication-agent-@{int}/** rwk,
 owner link @{user_cache_dirs}/polkit-kde-authentication-agent-@{int}/** -> @{user_cache_dirs}/polkit-kde-authentication-agent-@{int}/**,
```

Reply:
Thanks! So I'm going to modify it once more.